netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -515,6 +527,11 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
       }
       super.userEventTriggered(ctx, evt);
     }
+
+    private SslHandler sslHandler(ChannelPipeline pipeline) {
```

Comment:
nit: it doesn't seem necessary to create a trivial private method that has only one caller.